README.md

```diff
@@ -1,18 +1,32 @@
 # tdex-feeder
+
 Feeder allows to connect an external price feed to the TDex Daemon to determine the current market price
 
 ## ⬇️ Install
 
-TBD
+### Build 
```

Comment:
Here in install you should put the instructions for someone who wants to tun the compiled binary,

Example of doc here https://github.com/TDex-network/tdex-docs/blob/beta/tdex-daemon.md#install